Library/Homebrew/cmd/update.rb

```diff
@@ -57,6 +58,7 @@ def update
     else
       puts "Updated Homebrew from #{master_updater.initial_revision[0,8]} to #{master_updater.current_revision[0,8]}."
       report.dump
+      Descriptions.refresh_cache
```

Comment:
I'm worrying this would have huge impact on the performance. Especially considering`brew update` is already slow enough.